src/sso/sso.spec.ts

```diff
@@ -58,7 +61,7 @@ describe('SSO', () => {
 
           const url = workos.sso.getAuthorizationURL({
             domain: 'lyft.com',
-            projectID: 'proj_123',
+            clientID: 'proj_123',
```

Comment:
Should we be replacing the other pre-existing `projectID` parameters in this spec with `clientID` as well?

Reply:
Removed projectID from the specs. Definitely let me know if anything looks funky still! 🙏